### ISSUE TITLE
Parse Error: using hiding

### ISSUE THREAD

**Issue by leptonyu** [ndmitchell/hlint#628]
```Haskell
Error: Parse error
Found:
    import qualified Data.ByteString.Char8          as BC
    import           Data.Menshen
  > import           Data.Swagger                   hiding (get, pattern)
```

Another error is

```Haskell
do
  >   m  <- mobile ?: pattern "^1[0-9]{10}$"
  return ()
```

**Comment by ndmitchell** [comment]
It's actually with `pattern` - see https://github.com/ndmitchell/hlint/issues/216 for another report. I'll close this one in favour of that one.
